### PR TITLE
Add automated KPI forecast data to Looker

### PR DESCRIPTION
**Pull request by skahmann3**
Adding table with detailed view of forecast output and table with confidence interval output. Also moving this out of the model file so we have the appropriate view and explore files.

### CODE REVIEW

**Comment by skahmann3** [comment]
@irrationalagent -- Will you review this PR to add the automated KPI forecast data to Looker? I'm adding both the detailed table with the individual component trends and the confidence intervals table

File: kpi/views/Automated_KPI_Forecast_Confidence_Intervals.view.lkml [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    }
+
+  dimension: value {
+    hidden:  yes
+    sql: ${TABLE}.value ;;
+  }
+
+  measure: Estimated_Value {
+    type: number
+    description: "Forecasted if after forecast date, observed if before."
```
Comment (irrationalagent) [inline kpi/views/Automated_KPI_Forecast_Confidence_Intervals.view.lkml:39]: which value is this using if its after the forecast date? 50th percentile? Also, maybe re-phrase: "Forecast Date" to "Forecast Run Date" to match the label

File: kpi/views/Automated_KPI_Forecasts.view.lkml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      month,
+      month_name,
+      quarter,
+      year,
+    ]
+    convert_tz: no
+    datatype: date
+  }
+
+  dimension: forecast_date {
```
Comment (irrationalagent) [inline kpi/views/Automated_KPI_Forecasts.view.lkml:20]: "Forecast Run Date" to make consistent with the CIs?

**Review by irrationalagent — APPROVED** [review]
r+ wc...
would it make sense to join these two explores? or is there something im missing where they wouldn't line up nicely

File: kpi/views/Automated_KPI_Forecasts.view.lkml (hunk 2 above)
Comment (skahmann3) [reply kpi/views/Automated_KPI_Forecasts.view.lkml:20]: Fixed.

File: kpi/views/Automated_KPI_Forecast_Confidence_Intervals.view.lkml (hunk 1 above)
Comment (skahmann3) [reply kpi/views/Automated_KPI_Forecast_Confidence_Intervals.view.lkml:39]: Fixed the label. The value isn't the 50th percentile, it's the average of the uncertainty samples. (I've also added this to the label)

**Comment by skahmann3** [comment]
Thanks @irrationalagent! Merging is a bit tricky given the dates from the two datasets. `Automated KPI Forecasts` has the daily forecast data for QDAU and DAU and `Automated KPI Confidence Intervals` rolls up the daily data to monthly. I've updated the two explores to not allow users to switch to a date specification besides day (for the raw output) and month (for the rolled-up output) as we don't want Looker rolling this data up on its own

**Pull request merged**